addon/components/tf-fields/input-text.js

```diff
@@ -0,0 +1,34 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from '../../templates/components/tf-fields/input-text';
+import computed from 'ember-computed';
+import get from 'ember-metal/get';
+const { TextField } = Ember;
+
+/**
+  sizeGroup options: 'large', 'small', 'default'
+
+  @public
+  @class TfFieldsInputText
+  @extends Ember.TextField
+ */
+export default TextField.extend({
+  layout,
+
+  classNames: ['c-tf-fields__input-text', 'c-tf-fields'],
+  attributeBindings: ['aria-label', 'aria-describedby', 'role'],
+  classNameBindings: ['sizeCategory', 'disabled:c-tf-fields__input-text--disabled'],
+
+  sizeGroup: 'default',
+
+  sizeCategory: computed('sizeGroup', {
+    get() {
+      const sizeGroup = get(this, 'sizeGroup');
+      return `c-tf-fields__input-text--${sizeGroup}`;
```

Comment:
Not really an issue for this PR — but, I think this is a pattern we're going to see a bunch. And, I think we'll probably also want some consistent way of providing feedback to the consumer if they provide a not-supported option. So, if I decided to put `'x-large'`, there would be a warning to the console that says, `Only 'large', 'small', and 'default' are supported 'sideCategory' options` or something.